config/settings/base.py

```diff
@@ -343,12 +344,34 @@ def env(name, default=NoDefaultValue, type_=str):
 GITHUB_OAUTH_SCOPES.append("repo" if GITHUB_OAUTH_PRIVATE_REPO else "public_repo")
 
 # SF client settings:
-SF_CALLBACK_URL = env("SF_CALLBACK_URL", default=None)
+SFDX_CLIENT_CALLBACK_URL = env(
+    "SFDX_CLIENT_CALLBACK_URL", default=env("SF_CALLBACK_URL", default=None)
+)
+SFDX_CLIENT_ID = env("SFDX_CLIENT_ID", default=env("SF_CLIENT_ID", default=None))
+SFDX_CLIENT_SECRET = env(
+    "SFDX_CLIENT_SECRET", default=env("SF_CLIENT_SECRET", default=None)
+)
+SFDX_SIGNUP_INSTANCE = env(
+    "SFDX_SIGNUP_INSTANCE", default=env("SF_SIGNUP_INSTANCE", default=None)
+)
```

Comment:
(There was no reason this one needed to be renamed, since it was only used by Metecho and not CumulusCI, but whatever.)